### ISSUE TITLE
What is the relation between total_Active_users to the active_users ?

### ISSUE THREAD

**Issue by vineethj1729** [github-copilot-resources/copilot-metrics-viewer#51]
Whatever the data is available in the metrics result, the cumulative data is for every field is matching but not for active users.
For example: Let's take a day data result, in that breakdown field we have individual fields like suggestions, editors per active users 
But the cumulative active user's summation didn't match with total active users rest of the fields to total counts are matching.

**Comment by DevOps-zhuang** [comment]
Yes, it is true. since the 'active_users' in breakdown are classified per lanauge+edition combination. so If I use both typescript, and vue in vs code in one day, I will be counted in both 'active_users' of typesript+VScode, and typescript+Vue. the the sum of the 'active_users' wil be greater than the 'total_active_users'.

**Comment by vineethj1729** [comment]
Thank you @**DevOps-zhuang**. 

1. However, is there any way to find out the users who used copilot multiple times more than 1 time IDE/Editor or languages more than 1 time in a day. (**users name or user unique identification**)

2. Since we are trying to find the productivity of an individual users/ individual teams who contributing more to the organization using copilot.

**Comment by DevOps-zhuang** [comment]
> Thank you @**DevOps-zhuang**.
> 
> 1. However, is there any way to find out the users who used copilot multiple times more than 1 time IDE/Editor or languages more than 1 time in a day. (**users name or user unique identification**)
> 2. Since we are trying to find the productivity of an individual users/ individual teams who contributing more to the organization using copilot.

Thanks @vineethj1729  for your input. and as I understand, there are two ways to achieve your target, both takes some effort.

1) call the seat API more often in a day, like call it each 4 hours, so I will call 6 times each time, for active users, it will return several    records.

2) add a proxy between the copilot extension and github. then save all the traffic for analysis.

and GitHub is redesiging the teams support for the usage, hope it can be release soon, then at that time, We can get the team level usage.

**Comment by vineethj1729** [comment]
Thank you **@DevOps-zhuang** 
It means a lot😊
